src/static/lib/infusion/Version.md

```diff
@@ -1,19 +1,8 @@
-This version of Infusion was created using a custom build from the master branch:
+Uses the infusion-uiOptions-2.0.0-minified.zip custom Infusion 2.0 custom build provided on
+the [Infusion 2.0 release page](https://github.com/fluid-project/infusion/releases/tag/infusion-2.0).
 
-https://github.com/fluid-project/infusion/
-
-commit 809231c7525798ead15f214497721b6ea9e619f7
-
-using the command line:
-```
-    grunt custom --source=true --include="uiOptions" --name="uio"
-```
-
-The file src/framework/preferences/html/SeparatedPanelPrefsEditorFrame.html was modified to
-reference infusion-uio.js instead of individual js files.
-
-The following directories and files were stripped out of the build, since they contain
-code that is included in the infusion-uio.js file or is not required by this plugin:
+The following directories and files were stripped out of the build, because they contain
+code that is included in the infusion-uiOptions.js file or is not required:
```

Comment:
rephrase to: "The following directories and files were removed from the build, because they are already included in infusion-uiOptions.js file, or are no longer needed:"